deeplenstronomy/check.py

```diff
@@ -66,7 +66,9 @@ def __init__(self, full_dict, config_dict):
                                                                'GAUSSIAN_ELLIPSE_KAPPA', 'GAUSSIAN_ELLIPSE_POTENTIAL', 'MULTI_GAUSSIAN_KAPPA',
                                                                'MULTI_GAUSSIAN_KAPPA_ELLIPSE', 'INTERPOL', 'INTERPOL_SCALED', 'SHAPELETS_POLAR', 'SHAPELETS_CART',
                                                                'DIPOLE', 'CURVED_ARC', 'ARC_PERT', 'coreBURKERT', 'CORED_DENSITY', 'CORED_DENSITY_2',
-                                                               'CORED_DENSITY_MST', 'CORED_DENSITY_2_MST', 'NumericalAlpha', 'MULTIPOLE', 'HESSIAN']}
+                                                               'CORED_DENSITY_MST', 'CORED_DENSITY_2_MST', 'NumericalAlpha', 'MULTIPOLE', 'HESSIAN',
```

Comment:
just for convenience in the future, current lenstronomy versions do contain an array with all the model names here: https://github.com/sibirrer/lenstronomy/blob/93c8f1a3e7e7ac9a90adb70aaf8e8c29537ec51b/lenstronomy/LensModel/profile_list_base.py#L6

Reply:
Yes this part of the deeplenstronomy check.py module is a relic from before lenstronomy had all model names accessible. Issue https://github.com/deepskies/deeplenstronomy/issues/44 pertains to this.

In a separate pull request, someone can update check.py to look for lenstronomy's array of all the model names, but in this case I think we can just go ahead and merge the single model.